apps/dashboard/src/main/java/com/akto/action/testing/StartTestAction.java

```diff
@@ -543,6 +553,59 @@ public String fetchMetadataFilters(){
         return SUCCESS.toUpperCase();
     }
 
+    // this gives the count for test runs types{ All, CI/CD, One-time, Scheduled}
+    // needed for new ui as the table was server table.
+    public String computeAllTestsCountMap(){
+        Map<String,Long> result = new HashMap<>();
+
+        long totalCount = TestingRunDao.instance.getMCollection().estimatedDocumentCount();
```

Comment:
why estimatedDocumentCount, nd not the exact count?

Reply:
fixed